internal/slicer/slicer.go

```diff
@@ -235,38 +242,54 @@ func Run(options *RunOptions) (*Report, error) {
 		reader.Close()
 		packages[slice.Package] = nil
 		if err != nil {
-			return nil, err
+			return err
 		}
 	}
 
-	// Create new content not coming from packages.
-	done := make(map[string]bool)
+	// Create new content not coming from packages excluding GeneratePath(s).
+	// First group them by their relative path. Then create them and attribute
+	// them to the appropriate slices.
```

Comment:
It's not clear we want to do that, though. More on the code below.

Reply:
Resolving this comment because of https://github.com/canonical/chisel/pull/142#discussion_r1771655578.